.github/policies/resourceManagement.yml

```diff
@@ -4477,27 +4477,27 @@ configuration:
       - requestReview:
           reviewer: calvinhzy
       description: '[graph] Auto assign labels and reviewers based on PR title/description.'
-    - if:
-      - payloadType: Issues
-      - or:
-        - titleContains:
-            pattern: '\b([Ii][Nn][Ss][Tt][Aa][Ll][Ll])\b'
-            isRegex: True
-        - bodyContains:
-            pattern: '\b([Ii][Nn][Ss][Tt][Aa][Ll][Ll])\b'
-            isRegex: True
-      - or:
-        - isAction:
-            action: Opened
-      then:
-      - addLabel:
-          label: Auto-Assign
-      - addLabel:
-          label: Installation
-      - assignTo:
-          users:
-          - jiasli
-      description: '[install] auto assign labels and users based on issue description.'
+#    - if:

```

Comment:
The misjudgment rate of using the install keyword to match install-related issues is too high, so this task is temporarily disabled.

Reply:
So for issues really related to installation, we will not assign label automatically after this change?

Reply:
Yes, without using keywords, we have no better way to distinguish whether it is an install-related issues.